packages/php-wasm/web-service-worker/src/initialize-service-worker.ts

```diff
@@ -45,12 +45,6 @@ export function initializeServiceWorker(config: ServiceWorkerConfiguration) {
 				return;
 			}
 		}
-
-		console.debug(
-			`[ServiceWorker] Serving request: ${getRelativePart(
-				removeURLScope(url)
-			)}`
-		);
```

Comment:
@adamziel you mentioned that we should remove these logs. Let me know if you would like to keep some of them.

Reply:
This is great, thank you!